### PR TITLE
[new release] owee (0.5)

### PR DESCRIPTION
**Pull request by let-def**
OCaml library to work with DWARF format

- Project page: <a href="https://github.com/let-def/owee">https://github.com/let-def/owee</a>

##### CHANGES:

- Partial support for Dwarf-5 contributed by @billduff and @cgaebel

### CODE REVIEW

**Comment by let-def** [comment]
(The CI might run tests that only work with recent versions of GCC, I am curious to see on which OSes they pass)

**Comment by cgaebel** [comment]
Feel free to add an upper-bound to magic-trace's use of owee. We'll release a new version to opam soon.

File: packages/owee/owee.0.5/opam [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+bug-reports: "https://github.com/let-def/owee"
+license: "MIT"
+dev-repo: "git+https://github.com/let-def/owee.git"
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.06"}
+  "dune" {>= "1.0"}
```
Comment (mseri) [inline packages/owee/owee.0.5/opam:14]:
```suggestion
  "dune" {>= "2.0"}
```
According to the dune-project file

**Comment by mseri** [comment]
The CI failures are unrelated. Thanks!

**Pull request merged**